nautobot/core/authentication.py

```diff
@@ -106,12 +106,12 @@ def has_perm(self, user_obj, perm, obj=None):
         return False
 
 
-class DummyBackend(BaseBackend):
+class ExampleBackend(BaseBackend):
```

Comment:
I was curious about this one so I checked. It was added in b94452e61b, but it appears that the code that actually made use of it was removed by a33ccc1b6fc3e9b54c41b53350608d146411101b. I think we could/should delete this class, but I'm fine if you want to defer that to a different PR.

Reply:
Confirmed no usages found...
<img width="237" alt="Screenshot 2022-01-07 at 17 09 03" src="https://user-images.githubusercontent.com/655255/148580437-62453893-4093-4cb6-a5fd-425d4b18cda6.png">